crescent/internal/registry.py

```diff
@@ -171,11 +173,11 @@ def build_commands(self) -> Sequence[AppCommand]:
                     built_commands[key] = AppCommand(
                         name=unwrap(command.metadata.group).name,
                         description=unwrap(command.metadata.group).description or "No Description",
-                        type=command.metadata.app.type,
-                        guild_id=command.metadata.app.guild_id,
+                        type=command.metadata.app_command.type,
+                        guild_id=command.metadata.app_command.guild_id,
                         options=[],
-                        default_member_permissions=command.metadata.app.default_member_permissions,
-                        is_dm_enabled=command.metadata.app.is_dm_enabled,
+                        default_member_permissions=command.metadata.app_command.default_member_permissions,  # noqa
```

Comment:
im changing this

Reply:
Oh lol, how did I forget I should have used parentheses to separate the lines? My bad!